BUILD.readme

```diff
@@ -10,6 +10,18 @@ alwayseeded :
 	- GNU gettext
 	- GNU libiconv
 
+Simple Build in Linux without libcu8
+```bash
+git clone https://github.com/darkbatcher/picobat.git
+cd picobat
+sudo apt install gcc
+sudo apt install bison
+sudo apt install flex
+sudo apt install gettext
+make no-libcu8
```

Comment:
libcu8 isn't built by default, there is no need to explicitly disable it (unless we enabled it earlier)

Reply:
Anyways, telling those sudo apt install tools is good for developer-friendly documents.